src/content/1.7/development/webservice/tutorials/3-prestashop-webservice-lib/4-create-resource.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 4 - Create a resource
+weight: 4
+---
+
+# Create a resource
+
+### Creation workflow
+
+The creation workflow is a bit more complex than reading some data from the API, mainly because we rarely manage data via XML format. In most use cases the user is presented with a human understandable component, like a form, and the data entered is then processed. Besides we need to make sure that the XML sent to the webservice is understandable and complete.
+
+As we explained in a previous tutorial the webservice provides two [resource schemas](/1.7/development/webservice/tutorials/2-testing-access/#resource-schemas). For creation we are going to use the *blank* schema which is an empty representation of a resource. This empty XML will be filled with our data and then sent to the webservice using the `add()` method.
+
+{{< figure src="../../../img/create-resource.png" title="Update Resource via Webservice" >}}
+
+> You can update this schema using the [source XML file](/schemas/1.7/create-resource.xml) importable in services like [draw.io](https://draw.io).
```

Comment:
I'm not a fan of using quote blocks for notes, but I don't have a better suggestion...

Reply:
Maybe add all links at the bottom?

Reply:
I just followed how we integrated the schemas in other pages.. Should I change all of them? Or change all the pages in a dedicated PR?